cogs/generalCog.py

```diff
@@ -54,35 +58,34 @@ async def card(self, ctx, arg: str = commands.parameter(description="Name of rul
             elif os.path.isfile(file_path_png):
                 file_path = file_path_png
             else:
-                await ctx.reply("Rule card has wrong file ending... Abort.")
+                await ctx.reply(msg["card_wrong_type"])
             await ctx.reply(file=discord.File(file_path))
             if three_best:
-                await ctx.reply(f"Die drei besten Matches sind: {three_best}")
+                await ctx.reply(msg["card_best_matches"].format(best=three_best))
         else:
-            await ctx.reply("No such rule card...")
+            await ctx.reply(msg["card_not_found"])
 
-    @commands.command(help="Rolls dice.\n"
-                           "2d6+3: Sum the result of 2 6-sided dice and 3.\n"
-                           "2@3d20: Roll 3d20 and take the second highest i.e. (20,15,5) => 15.\n"
-                           "Special rolls:\n"
-                           "I: 1d20, Io: 1@2d20, Ioo 1@3d20\n"
-                           "III: 2@3d20, III': 2@4d20, III'' 2@5d20\n", aliases=['w'])
-    async def r(self, ctx, roll: str = commands.parameter(default="III", description="Dice string to parse."),
+    @commands.command(help=msg["r_help"], aliases=['w'])
+    async def r(self, ctx, roll: str = commands.parameter(default="III", description=msg["r_desc"]),
                 identifier: str = commands.parameter(default="", description="Identifier.")):
         roll = roll.replace(" ", "")
         for key, value in NAMED_ROLLS:
             roll = roll.replace(key, value)
         total_result_str, total_result = parse_die.parse_roll(roll)
-        total_result = f"""```md
-# {ctx.author.nick or ctx.author.global_name} --- {identifier} {total_result}
-Details: {total_result_str}```"""
+        total_result = msg["r_result"].format(
+            author=ctx.author.nick or ctx.author.global_name,
+            identifier=identifier,
+            result=total_result,
+            details=total_result_str
```

Comment:
this should be `total_result_str=total_result_str` according to messages.yml or you need to change it there